custom-elements.json

```diff
@@ -0,0 +1,271 @@
+{
+  "schemaVersion": "1.0.0",
+  "readme": "",
+  "modules": [
+    {
```

Comment:
Should we should exclude extension-related scripts and only include actual custom element modules?

Reply:
Fixed